CONTRIBUTING.md

```diff
@@ -0,0 +1,99 @@
+# Guide to Making Developer Contributions
+
+## Overview:
+
+A CSV with package name and version is uploaded by the user, 
+then the risk of the packages are assessed using `riskmetric`.
+[TODO this is shakey for me.... where is this function actually called... 
+I see where we write comments to the database, 
+and where we read fields FROM the database,
+but where and how are the database's risk metrics written?]
+
+The database (SQLite by default) is updated to include the user specified package 
+and its associated metrics. 
+
+These metrics are used to populate the UI of the application. 
+Each package only needs to be entered into the database once for the user
+to see its associated risk.
+
+## Package files and folder 
+
+[TODO order in which to introduce these files, 
+maybe also include a schematic of how the files are related]
+
+### Root
+- `loggit.json` [TODO: log of user collected data within the app?]
```

Comment:
A side note: this file will be removed from the repo. So it will exist side by side with the app (or better yet, we can let the user decide where to save it), but it won't be committed to the repo.

The `loggit.json file` should serve as a log file, meaning it 'should' contain information about the uploaded packages, errors, and anything else we consider relevant. Right now, it logs errors as well as info when file is uploaded. By doing a `grep`, I can see the function that writes to this file, i.e. `loggit`, is used here:

```
./Modules/dbupload.R:        loggit("ERROR", paste("Error in extracting general info of the package", package_name, "info", e), app = "fileupload-webscraping")
./Modules/dbupload.R:      loggit("ERROR", paste("Error in uploading the general info of the package", package_name, "info", e), app = "fileupload-DB")
./Modules/dbupload.R:      loggit("ERROR", paste("Error in extracting cum metric info of the package:", package_name, "info", e), app = "fileupload-webscraping")
./Server/sidebar.R:  loggit("INFO", paste("decision for the package", values$selected_pkg$package, 
./Server/uploadpackage.R:  loggit("INFO", paste("Summary of the uploaded file:",input$uploaded_file$name, 
```